src/lib/clients/rest.ts

```diff
@@ -238,30 +238,57 @@ export class RestClient implements REST {
     return this.fetchJson(`/get_profile?account=${address}`)
   }
 
-  public async getPosition(params: types.MarketAndAddressGetterParams) {
-    if (!params.address && !this.wallet) {
-      throw new Error('get_account: missing address param')
-    }
-    let address = ''
-    if (!params.address) {
-      address = this.wallet.pubKeyBech32
-    } else {
-      address = params.address
+  public async getPosition(params: types.GetPositionGetterParams) {
+    if (!params.positionId) {
+      throw new Error('position_id: missing position_id param')
     }
-    return this.fetchJson(`/get_position?account=${address}&market=${params.market}`)
+    return this.fetchJson(`/get_position?position_id=${params.positionId}`)
```

Comment:
avoid regression where existing functionality is removed. we should still support adding address/market params if provided.